tests/cli-rustup.rs

```diff
@@ -856,6 +856,18 @@ fn multirust_dir_upgrade_rename_multirust_dir_to_rustup() {
     });
 }
 
+#[test]
+fn rustup_from_non_existing_path() {
+    setup(&|config| {
+        let non_existing_dir = config.homedir.join(".remove_me");
+        let mut cmd = clitools::cmd(config, "rustup", &["-h"]);
+        cmd.env_remove("CARGO_HOME");
+        cmd.current_dir(non_existing_dir);
```

Comment:
I'm tryint to write a basic test here, but seems like `cmd` is checking the directory prior to executing the command and so failing with a no such file or directory. Any ideas on how to test this out?